### DISCUSSION TITLE
How to create seemless pattern?

### DISCUSSION THREAD

**Discussion by GodwinF** [lllyasviel/Fooocus#978] [Q&A]
I want to create a seemless pattern so i tried 'SAI Texture'. It created a pattern but it isn't seemless.
So is there any way I can create a seemless pattern?

Here's just an example, with the promot 'flower' and the style 'SAI Texture'.
![Clipboard Image](https://github.com/lllyasviel/Fooocus/assets/102510013/6832e62b-20d5-40fd-8ae0-8acfac99f766)

**Comment by binaryatlas** [comment]
any answer on this?
